### PR TITLE
Get current user permission and add more return properties to SearchResult

### PR DESCRIPTION
**Pull request by VanLePham**

### CODE REVIEW

File: BaseSpace.SDK/ServiceModels/Request/GetUserPermissionRequest.cs [hunk 1]
```diff
@@ -0,0 +1,14 @@
+﻿
+
+namespace Illumina.BaseSpace.SDK.ServiceModels
+{
+    public class GetUserPermissionRequest : AbstractResourceRequest<GetUserPermissionResponse>
```
Comment (maximmass) [inline BaseSpace.SDK/ServiceModels/Request/GetUserPermissionRequest.cs]: perhaps rename this and the client method to something like GetAccessTokenDetails ?

Comment (VanLePham) [inline BaseSpace.SDK/ServiceModels/Request/GetUserPermissionRequest.cs]: Hi,

Do you want me to rename the UserPermission class to something like AccessTokenDetails  as well?

[DataContract]
public class UserPermission : AbstractResource

Thanks,
Van

**Comment by maximmass** [comment]
lgtm. go ahead and merge.

**Comment by VanLePham** [comment]
I don't think I have write permission to merge the request

**Pull request merged**